make-uniform/production/Build Standard Database Functions.R

```diff
@@ -63,6 +63,64 @@ get_nearest_station <- function(station_names_df, survey_records_df, operator_ke
 }
 
 
+# get_rail_names <- function(station_names_shp, 
```

Comment:
@yuqiww, please remind me. This function was deprecated because rail is now handled elsewhere in the code?

Reply:
Yes. Please see the email titled "MTC Onboard Survey Scripts" from John Helsel on 2/19/2020.